flutter/news/lib/features/frontpage/data/repositories/articles_repository_impl.dart

```diff
@@ -1,19 +1,46 @@
 import 'package:dartz/dartz.dart';
-import 'package:news/core/error/failures.dart';
-import 'package:news/features/frontpage/domain/entities/article.dart';
-import 'package:news/features/frontpage/domain/repositories/articles_repository.dart';
+import 'package:news/core/error/exceptions.dart';
+
+import '../../../../core/error/failures.dart';
+import '../../../../core/network/network_info.dart';
+import '../../domain/entities/article.dart';
+import '../../domain/repositories/articles_repository.dart';
+import '../datasource/articles_data_source.dart';
+import '../datasource/articles_local_data_source.dart';
 
 class ArticlesRepositoryImpl implements ArticlesRepository {
```

Comment:
I wonder why having an `abstract` articlesRepository if we only have one implementation of it.

Doesn't sound to you like extra code that we don't need, for now?

Reply:
updated on https://github.com/novoda/multiplatform-playground/commit/8323a52c3c021a85d1662fa2609c061534a5fce3